pkgs/development/python-modules/cirq/default.nix

```diff
@@ -48,6 +48,15 @@ buildPythonPackage rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "freezegun~=0.3.15" "freezegun" \
+      --replace "matplotlib~=3.0" "matplotlib" \
+      --replace "networkx~=2.4" "networkx" \
+      --replace "numpy~=1.16, < 1.19" "numpy" \
```

Comment:
there was a numpy abi change, this might be intentional

Reply:
Fair, but this isn't the right format anyways so it doesn't do anything (should be ``numpy~=1.16,<1.19``) and I tested it working against master with ``numpy==1.19.1``. I'm OK removing the line, but I'm just trying to avoid as many version conflicts as possible so I don't have to update this package every time matplotlib/networkx/numpy update.